.github/workflows/ci_push.yml

```diff
@@ -57,13 +57,10 @@ jobs:
                           --build-arg REMOTE=ghcr.io/nasa
                           -t ghcr.io/${{ github.repository_owner }}/isaac:msgs-ubuntu20.04
 
-    # Temp disabling the analyst build because it has a version conflict issue. Re-enable once
-    # #136 is fixed.
-    # *** Remember to uncomment the push below, as well!
```

Comment:
Missed this! Refers to: https://github.com/nasa/isaac/blob/f232c0677bfe213c47b247542df544993d0823fc/.github/workflows/ci_push.yml#L75

Reply:
awesome, thank you!